xbmc/addons/AddonInstaller.h

```diff
@@ -56,6 +56,12 @@ enum class InstallModalPrompt
   NO_PROMPT,
 };
 
+enum class MigrationJob
```

Comment:
This might be better called `MigrationActive` as it is is used not only in `DoInstall` which is job specific but also in `InstallAddonUpdates` and `InstallAddons`.

Reply:
Or even be made more generic and not implementation specific. Migration is only a possible use of the add-on updates method, it should then be agnostic to what is calling it. Maybe CheckForUpdates:YES/NO?

Please also make sure doxygen stays in sync with the new parameters.

Reply:
Or even `AllowCheckForUpdates::YES/NO`.

Reply:
ok, let's use `AllowCheckForUpdates`, also removed the default argument and doxy updated

Reply:
I think you have this inverted @howie-f. You need to check the `YES`/`NO`'s